src/msw/headerctrl.cpp

```diff
@@ -1136,16 +1142,16 @@ void wxHeaderCtrl::SetWindowStyleFlag(long style)
 
     // Update the native control style.
     long flags = m_nativeControl->GetWindowStyleFlag();
-    flags = ApplyHeaderReorderFlagToStyle(flags);
-    m_nativeControl->SetWindowStyleFlag(flags);
-}
 
-long wxHeaderCtrl::ApplyHeaderReorderFlagToStyle(long style)
-{
-    if ( HasFlag(wxHD_ALLOW_REORDER) )
-        return style | wxHD_ALLOW_REORDER;
+    flags &= ~wxHD_ALLOW_REORDER;
+    if (HasFlag(wxHD_ALLOW_REORDER))
+        flags |= wxHD_ALLOW_REORDER;
```

Comment:
Very minor, but I'd write this (and the lines below) as
```
if ( HasFlag(wxHD_FOO) )
    flags |= wxHD_FOO;
else
    flags &= ~wxHD_FOO;
```
not because of any efficiency considerations, but just because it seems much more clear what this does to me.